p2panda-js/src/wasm-adapter/node.ts

```diff
@@ -0,0 +1,7 @@
+type WasmNode = typeof import('wasm-node');
+
+const wasm: Promise<WasmNode> = import('wasm-node').then((lib) => {
+  return lib;
+});
+
+export default wasm;
```

Comment:
This whole thing feels like misdirection to me. Why not just import `wasm-node` from `index.ts` here?

Reply:
Yes, right! I had it explicitly using the Promise before to have more control over what should get imported (there is actually loads of "garbage" in `lib` which gets public if we do it like this, originally I only deconstructed `KeyPair` to prevent this. For now we can remove this and clean up later, think this should be connected to the TypeScript stuff anyways.